DemoApp/Assets/Scripts/HomeScreen.cs

```diff
@@ -1,18 +1,234 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using UnityEngine;
+using UnityEngine.UI;
 
 public class HomeScreen : MonoBehaviour
 {
-    // Start is called before the first frame update
+    private const string MaxSdkKey = "ENTER_MAX_SDK_KEY_HERE";
+    private const string InterstitialAdUnitId = "ENTER_INTERSTITIAL_AD_UNIT_ID_HERE";
+    private const string RewardedAdUnitId = "ENTER_REWARD_AD_UNIT_ID_HERE";
+    private const string BannerAdUnitId = "ENTER_BANNER_AD_UNIT_ID_HERE";
+
+    public Button showInterstitialButton;
+    public Button showRewardedButton;
+    public Button showBannerButton;
+    public Button mediationDebuggerButton;
+    public Text interstitialStatusText;
+    public Text rewardedStatusText;
+
+    private bool isBannerShowing = false;
+
     void Start()
     {
-        
+        showInterstitialButton.onClick.AddListener(ShowInterstitial);
+        showRewardedButton.onClick.AddListener(ShowRewardedAd);
+        showBannerButton.onClick.AddListener(ToggleBannerVisibility);
+        mediationDebuggerButton.onClick.AddListener(MaxSdk.ShowMediationDebugger);
+
+        MaxSdkCallbacks.OnSdkInitializedEvent += (MaxSdkBase.SdkConfiguration sdkConfiguration) =>
+        {
+            // AppLovin SDK is initialized, configure and start loading ads
+            Debug.Log("MAX SDK Initialized");
+
+            if (sdkConfiguration.ConsentDialogState == MaxSdkBase.ConsentDialogState.Applies)
+            {
+                // Show user consent dialog
+            }
+            else if (sdkConfiguration.ConsentDialogState == MaxSdkBase.ConsentDialogState.DoesNotApply)
+            {
+                // No need to show consent dialog, proceed with initialization
+            }
+            else
+            {
+                // Consent dialog state is unknown. Proceed with initialization, but check if the consent
+                // dialog should be shown on the next application initialization
+            }
+
+            MaxSdk.SetHasUserConsent(true);
+            MaxSdk.SetIsAgeRestrictedUser(false);

```

Comment:
No no no.

Reply:
If pubs just set to have user consent for all their traffic, they can point to our example and we can be liable.